assets/speakerEmailTemplate.md

```diff
@@ -0,0 +1,43 @@
+Hello {name},
+
+Thank you for agreeing to speak at JSOxford! We are very much looking forward to you speaking on the {date}. I have collected some information for you about the venue and technical things. 
+
+Venue
+We will be holding JSOxford at The Story Museum.
+https://www.google.co.uk/maps?q=the+story+museum&biw=1280&bih=647&um=1&ie=UTF-8&sa=X&ved=0ahUKEwiMo6_nx63YAhWKI8AKHQxCBjsQ_AUICigB
+
+How to get there
+
+From the train station
+https://www.google.co.uk/maps/dir/Oxford+Railway+Station,+Park+End+Street,+Oxfordshire,+Oxford/The+Story+Museum,+Rochester+House,+42+Pembroke+St,+Oxford+OX1+1BP/@51.7521504,-1.2683431,16z/data=!3m1!4b1!4m14!4m13!1m5!1m1!1s0x4876c6a2790faf05:0xba97a9d53201a3c!2m2!1d-1.2699454!2d51.7534694!1m5!1m1!1s0x4876c6afffdb9abf:0x37007a572c93207d!2m2!1d-1.257986!2d51.750698!3e2
+
+From the bus station
+https://www.google.co.uk/maps/dir/Oxford+(Bus+Station),+Oxford/The+Story+Museum,+Rochester+House,+42+Pembroke+St,+Oxford+OX1+1BP/@51.7522993,-1.2618879,17z/data=!3m1!4b1!4m14!4m13!1m5!1m1!1s0x4876c6a4434a3237:0xe8855286b4e428a!2m2!1d-1.262528!2d51.753797!1m5!1m1!1s0x4876c6afffdb9abf:0x37007a572c93207d!2m2!1d-1.257986!2d51.750698!3e2
+
+From a few car parks
```

Comment:
Include park and ride (seacourt and redbirdge) - https://www.oxfordshire.gov.uk/cms/public-site/park-and-ride